WordPress/twentytwentyfive
# Add: Services - Two-column boxed pricing table

**Description**

Fixes https://github.com/WordPress/twentytwentyfive/issues/78

- Adding the two-column pricing pattern.
- I haven't used the "ticks" for the pricing plan details, but instead I used a regular list. Not sure if we want to pursue the "tick", I believe that's not possible out of the box.
- I'm also fixing the margin for the "Three column services" pattern that I noticed now that we have section styles.
- I'm also adjusting the button font size if we decide not to proceed with [this](https://github.com/WordPress/twentytwentyfive/pull/157).



**Screenshots**



https://github.com/user-attachments/assets/214457ef-4107-40b5-b741-b2bbfe114f5e


**Testing Instructions**

1. Create a page.
2. Add the pattern.
3. Confirm that the design is OK (spacing, border-color, font-size, etc).
